packages/react/src/components/radio-common/types.ts

```diff
@@ -0,0 +1 @@
+export type RadioGroupValue = string | number;
```

Comment:
Looks like we don't need RadioGroupValue in the React package anymore.
 Because we already move it to the 'components' package in [this PR](https://github.com/DevExpress/DevExtreme/pull/23450)
 
 So, let's delete this file :)

Reply:
You're right, I forgot to delete it. Fixed.